docs/visualize/tsvb.asciidoc

```diff
@@ -0,0 +1,115 @@
+[[TSVB]]
+== Visualizing your data with TSVB
+
+TSVB is a time series data visualizer that allows you to use the full power of the 
+Elasticsearch aggregation framework. With TSVB, combine an infinite 
```

Comment:
"With TSVB, combine an infinte..."  I wonder if you can soften it a bit with "With TSVB, you can combine an infinite....  

What's the difference between "aggregations" and "pipeline aggregations"

Is "in a meaningful way" needed?